JapaneseFilter/sections/antiadblock.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! These rules are against all kinds of ad blocking reinjection/circumvention scripts
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/65807
+||freemen.jp/wp-content/uploads/$script,~third-party
```

Comment:
This is kinda a wide rule, just specify it like this `||freemen.jp/wp-content/uploads/gftpkzd.js` or via Regex for now.
Also you can additionally add these rules. It can be solved via scriptlet, but I don't think that we need it now.
```
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=freemen.jp
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=freemen.jp
```

Reply:
Done.